timelock-impl/src/main/java/com/palantir/atlasdb/timelock/clock/ClockSkewMonitor.java

```diff
@@ -53,7 +54,7 @@ public static ClockSkewMonitor create(MetricsManager metricsManager,
         Map<String, ClockService> clocksByServer = Maps.toMap(
                 remoteServers,
                 (remoteServer) -> AtlasDbHttpClients.createProxy(metricsManager.getRegistry(),
-                        optionalSecurity, remoteServer, ClockService.class));
```

Comment:
@gokcanozakdag do we still need this `ClockSkewMonitor`? If not, we should remove most of the skew/leadership lease related code since it's not going to be used + makes refactors easier (don't have to think about whether or not it's safe to remove code etc.)

Reply:
I think it is time to remove `ClockSkewMonitor` 🔥 🔥

Reply:
Killed with fire